ui/StatusQ/src/StatusQ/Components/StatusCard.qml

```diff
@@ -306,23 +377,33 @@ Rectangle {
             }
             PropertyChanges {
                 target: secondaryLabel
-                color: disabled ? Theme.palette.directColor5: Theme.palette.primaryColor1
+                color: disabled ? sensor.containsMouse ?
+                                      Theme.palette.primaryColor1 :
+                                      Theme.palette.directColor5: Theme.palette.primaryColor1
             }
             PropertyChanges {
                 target: secondaryLabel
                 visible: !advancedMode && secondaryLabel.text
             }
             PropertyChanges {
                 target: secondaryLabel
-                text: disabled ? disabledText : secondaryText
+                text: disabled ? sensor.containsMouse ? root.enableText : disabledText : secondaryText
+            } 
+            PropertyChanges {
+                target: secondaryLabel
+                font.weight: disabled && sensor.containsMouse ? Font.Medium : Font.Normal
             }
             PropertyChanges {
                 target: tertiaryText
                 color: Theme.palette.directColor5
             }
             PropertyChanges {
                 target: tertiaryText
-                visible: tertiaryText.text
+                visible: advancedMode ? tertiaryText.text : (!sensor.containsMouse && tertiaryText.text) || disabled
```

Comment:
When I hover and the description text disappears, it looks odd in the sandbox — wondering why this reasoning for hovering to control the visibility of the main items?